src/Services/Virtualization/PortalAuthenticationModule.cs

```diff
@@ -20,158 +21,80 @@
 using SenseNet.Diagnostics;
 using SenseNet.Portal.Handlers;
 using System.Threading;
+using Newtonsoft.Json;
 using SenseNet.Configuration;
 using SenseNet.Security;
+using SenseNet.TokenAuthentication;
 
 namespace SenseNet.Portal.Virtualization
 {
-    internal class PortalAuthenticationModule : IHttpModule
+    public class PortalAuthenticationModule : IHttpModule

```

Comment:
I did not want to use this attribute, so I made it public to easily access it from any test.